spec/models/user_spec.rb

```diff
@@ -0,0 +1,55 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  provider               :string           default("email"), not null
+#  uid                    :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  allow_password_change  :boolean          default(FALSE)
+#  remember_created_at    :datetime
+#  confirmation_token     :string
+#  confirmed_at           :datetime
+#  confirmation_sent_at   :datetime
+#  unconfirmed_email      :string
+#  name                   :string
+#  nickname               :string
+#  image                  :string
+#  email                  :string
+#  tokens                 :json
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+require 'rails_helper'
+
+describe User, type: :model do
+  describe 'is valid' do
```

Comment:
Check out [shoulda-matchers gem](https://github.com/thoughtbot/shoulda-matchers). This gem will allow you to test validations with less code.